orion.py

```diff
@@ -11,8 +12,13 @@
 import pandas as pd
 
 from fmatch.matcher import Matcher
-from utils.orion_funcs import run_hunter_analyze, get_metadata, \
-                                set_logging, load_config, get_metric_data
+from utils.orion_funcs import (
```

Comment:
thoughts on just importing the whole file, so you dont have to specify every function as we continue to add